### PR TITLE
Remove comments in the SQL statement

### PR DESCRIPTION
**Pull request by sneaxiy**
Remove single line (--) and multiple line (/**/) comments in the SQL statement before running.

### CODE REVIEW

File: go/parser/sqlflow_parser_test.go [hunk 1]
```diff
@@ -308,3 +308,35 @@ func TestParserErrorMessage(t *testing.T) {
...
+
+	testFunc(`SELECT * FROM a;`, `SELECT * FROM a;`, true)
+	testFunc(`SELECT * FROM a -- I am a comment`, `SELECT * FROM a `, true)
+	testFunc("SELECT * FROM a -- I am a comment\n TO TRAIN b WITH -- 2nd comment\n INTO c",
+		"SELECT * FROM a \n TO TRAIN b WITH \n INTO c", true)
+	testFunc("SELECT '--' FROM a -- I am a comment\n TO TRAIN b WITH -- 2nd comment\n INTO c",
+		"SELECT '--' FROM a \n TO TRAIN b WITH \n INTO c", true)
+	testFunc("SELECT \"abc -- FROM a -- I am a comment\n TO TRAIN b WITH -- 2nd comment\n\" INTO c",
+		"SELECT \"abc -- FROM a -- I am a comment\n TO TRAIN b WITH -- 2nd comment\n\" INTO c", true)
+	testFunc("SELECT \"abc\\\" -- comment\" FROM b", "SELECT \"abc\\\" -- comment\" FROM b", true)
```
Comment (typhoonzero) [inline go/parser/sqlflow_parser_test.go:331]: Field names are not allowed to be like `abc\" -- comment`, this test case may be confusing, maybe change to `select * from t where f="...."`

**Review by typhoonzero — APPROVED** [review]
LGTM

File: go/parser/sqlflow_parser_test.go (hunk 1 above)
Comment (sneaxiy) [reply go/parser/sqlflow_parser_test.go:331]: The `abc\" -- comment` here is not a field name but a string literal.

I have tried this SQL statement (`SELECT "abc\" --comment"`) in MySQL, and it works well.

```
mysql> SELECT "abc\" --comment";
+----------------+
| abc" --comment |
+----------------+
| abc" --comment |
+----------------+
1 row in set (0.00 sec)
```

Comment (typhoonzero) [reply go/parser/sqlflow_parser_test.go:331]: `SELECT "abc\" --comment"` is OK yet `SELECT "abc\" --comment" FROM sometable` may cause error.

Comment (sneaxiy) [reply go/parser/sqlflow_parser_test.go:331]: `SELECT "abc\" --comment" FROM iris.train` is also OK in MySQL.

```
mysql> SELECT "abc\" --comment" FROM iris.train LIMIT 2;
+----------------+
| abc" --comment |
+----------------+
| abc" --comment |
| abc" --comment |
+----------------+
2 rows in set (0.00 sec)
```

Comment (typhoonzero) [reply go/parser/sqlflow_parser_test.go:331]: OK, cool.

**Pull request merged**